### PR TITLE
Feat/add notebook tutorial

### PR DESCRIPTION
**Pull request by WilliGautier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,10 +135,10 @@ docker run --rm \
 ```
 
 - Launch spark-shell with ARLAS-PROC dependency
-```
+```bash
 docker run -ti \
        -w /opt/work \
-       -v ${PWD}/data/ais:/opt/data \
+       -v ${PWD}/data/ais/data:/opt/data \
```
Comment (sfalquier) [inline README.md]: To be fair, this must be done in the same commit where you move the data directory. BTW, it's not a big issue in that context.

File: README.md [hunk 2]
```diff
@@ -120,6 +120,35 @@ Specify ARLAS-proc dependency in the dependencies section of your build.sbt file
...
+       -v $HOME/.ivy2:/home/jovyan/.ivy2 \
+       -p "8888:8888" \
+       almondsh/almond:0.9.1-scala-2.12.10
+```
+
+Open the link proposed in terminal to open Jupyter Notebook in a browser: `http://127.0.0.1:8888/?token=...`
+
+![Jupyter](./data/ais/images/notebook.png)
+
+Open `demo_notebook.ipynb` to run the tutorial notebook.
```
Comment (sfalquier) [inline README.md:150]: To be fair, this must be updated in the same commit where you add the notebook (or else, someone cherry picking this commit won't have the notebook working). BTW, not a big issue in that context.

**Review by sfalquier — APPROVED** [review]
Ok
And do not forget you have a glossary to produce for ARLAS-proc concepts.

**Comment by WilliGautier** [comment]
Better like this?

**Pull request merged**